Resources/public/js/cookie_consent.js

```diff
@@ -37,6 +37,8 @@ document.addEventListener("DOMContentLoaded", function() {
                 xhr.open('POST', cookieConsentForm.action);
                 xhr.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded');
                 xhr.send(serializeForm(cookieConsentForm, event.target));
+		// Clear all styles from body to prevent the white margin at the end of the page
+		document.getElementsByTagName("body")[0].style = null;
```

Comment:
```suggestion
		document.body.style.marginBottom = null;
		document.body.style.marginTop  = null;
```

It's the margins on line 11 and 17 you want to remove, and the element, and it should be available without using `getElementsByTagName`

Reply:
This is indeed better than my code only the line `document.body.style.marginTop  = null;` is not needed.
The line `document.body.style.marginBottom = null;` fixed the issue. Can you change it for me @NiklasBr?

Reply:
Well, from my perspective it is needed, because some people have the cookie consent on top, others at bottom, see lines 11 and 17